packages/orchestration/src/exos/chainAccountKit.js

```diff
@@ -160,6 +160,8 @@ export const prepareChainAccountKit = zone =>
           this.state.remoteAddress = remoteAddr;
           this.state.localAddress = localAddr;
           this.state.chainAddress = harden({
+            // FIXME need a fallback value like icacontroller-1-connection-1 if this fails
```

Comment:
```suggestion
            // FIXME need a fallback value like icacontroller-1-connection-1 if this fails.
            // Or, we should throw since a consumer can't do much w/o an address.
```